pkg/flink/handler.go

```diff
@@ -151,7 +158,7 @@ func (h flinkResourceHandler) OnAbort(ctx context.Context, tCtx pluginsCore.Task
 	return abortBehavior, nil
 }
 
-func flinkClusterTaskLogs(ctx context.Context, flinkCluster *flinkOp.FlinkCluster) ([]*core.TaskLog, error) {
+func FlinkClusterTaskLogs(ctx context.Context, flinkClusterName string, flinkClusterNamespace string) ([]*core.TaskLog, error) {
```

Comment:
I think it would still be better to pass in the FlinkCluster here.

Reply:
Currently trying to depend on this library for another flyte webApi plugin.
For some flyte plugins, we might not even have a `FlinkCluster` object, but only metadata about a FlinkCluster.

Reply:
Maybe it's better to come with a type here to encapsulate more metadata in the future if needed

Reply:
Yes. Makes sense. 
Will do it next time we add any more information to taskLogs. 👍